enhancements/machine-api/cluster-api-integration.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Cluster API Integration
+authors:
+  - "@JoelSpeed"
+  - "@alexander-demichev"
+reviewers:
+  - "@elmiko"
+  - "@Fedosin"
+  - "@lobziik"
+  - "asalkeld"
+  - "hardys"
+approvers:
+  - TBD
+creation-date: 2021-09-16
+last-updated: 2021-09-16
+status: implementable
+---
+
+# Cluster API Integration
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement describes the process of integrating the upstream [Cluster API](https://github.com/kubernetes-sigs/cluster-api)
+project into OpenShift standalone clusters.
+
+## Motivation
+
+We would like to give users the ablility to use Cluster API for machine management, as an alternative and possible future replacement for Machine API.
+
+### Goals
+
+- Run Cluster API controllers for managing infrastructure in a similar way to Machine API
+- Provide forward compatibility between Machine API (MAPI) and Cluster API (CAPI)
+- Ensure feature parity between MAPI and CAPI before migration
+
+### Non-Goals
+
+- Deprecate or remove any existing APIs
+- Stop providing support for Machine API in near future
+
+## Proposal
+
+In the longer term, our goal is to reconverge with the upstream Cluster API project and leverage the community built providers for managing Machines within OpenShift standalone clusters.
+To gather feedback on the feasability of this goal, we will first introduce Cluster API alongside Machine API as a technical preview, and gather feedback on Cluster APIs feasability as a
+replacement for Machine API.
+
+### User Stories
+
+#### Story 1
+
+As an openshift developer, I would like to leverage the upstream providers and reduce the barrier to OpenShift of supporting new platforms.
+
+#### Story 2
+
+As an openshift developer, I would like to gain the support of third parties who already have vested interests in maintaining Machine providers for various platforms and environments and benefit from their provider specific expertise as I try to add new features.
+
+#### Story 3
+
+As a cloud developer, I would like to easily onboard new platforms as this process is well documented and any implementation will be able to be leveraged by both Kubernetes and OpenShift customers, increasing the value of implementing a new provider.
```

Comment:
i might mention the CAPI community in here just to make it clear that we are using that community's documentation and guidance to create those new infrastructure providers.